src/reservationModal/reservations.js

```diff
@@ -0,0 +1,141 @@
+import PullMoviesData from './pullMovies.js';
+import FetchReservations from './fetchReservations.js';
+
+export default class Reservations {
+  constructor() {
+    this.body = document.querySelector('body');
+    this.viewReservationsBtns = document.getElementsByClassName('viewReservations');
+    this.fetchReservations = new FetchReservations();
+    this.reservationCount = 0;
+    this.movieID = 0;
+
```

Comment:
You did a good job working on this milestone! 💯 However, there are still some issues that need to be fixed in order to mark the project as approved:

- [ ] I noticed there is a bug with the creation of a new reservation:

[Screencast from 08-07-23 11:57:51.webm](https://github.com/geekelo/kingflix/assets/77069011/e84d2a67-62b1-4094-b197-58ce928971b8)

When I click on "reserve" the page reloads and the reservation is not persisted to the server. Please consider fixing this behavior! 🤓 🔧 

- [ ] I Also noticed a bug with the counters, when I open a movie that has, for example, 10 reservations and then open another one that has 0, the number 10 persists. Then, If I open a different one with x reservations, the number is 0:

[Screencast from 08-07-23 12:01:05.webm](https://github.com/geekelo/kingflix/assets/77069011/7af62a56-bfeb-45e2-a76c-eb3bdf6d0121)

Please consider looking into this! 💯